packages/rum-core/package.json

```diff
@@ -4,7 +4,11 @@
   "description": "Elastic apm core",
   "license": "MIT",
   "main": "src/index.js",
-  "repository": "https://github.com/elastic/apm-agent-js-base/tree/master/packages/rum-core",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/elastic/apm-agent-js-base.git",
```

Comment:
Isn't just using `https://...` simpler?

Reply:
yeah it is, Its just a preference that it is indeed a git url. I am fine with modifying it as well.

Reply:
Yeah, let's just use the `https:`